examples/dynamic-dao-implementation/README_pt.adoc

```diff
@@ -0,0 +1,379 @@
+:index-group: Proxy Beans
+:jbake-type: page
+:jbake-status: status=published
+= Implementação dinâmica de DAO
+
+Muitos aspectos dos Data Access Objects (DAOs) são muito repetitivos e
+chapa de ebulição. Como um recurso divertido e experimental, o TomEE suporta
```

Comment:
In this case, it seems better not to translate and keep the words in english "boiler plate" instead of "chapa de ebulição"